superset-frontend/src/dashboard/components/nativeFilters/FilterConfigModal.tsx

```diff
@@ -0,0 +1,283 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { findLastIndex, uniq } from 'lodash';
+import shortid from 'shortid';
+import { DeleteFilled } from '@ant-design/icons';
```

Comment:
You previously exported this module from `/src/common/comoonents`.... might be better to import it from there so we can keep monkeying around with it :)

Reply:
The DeleteFilled module? I don't think it's exported from there.

Reply:
Hmm... I thought all the Antd Icons were exported from there. I'll take a sweep at all this stuff later. Moving right along for now.